Formula/nuspell.rb

```diff
@@ -6,19 +6,28 @@ class Nuspell < Formula
 
   depends_on "catch2" => :build
   depends_on "cmake" => :build
+  depends_on "p7zip" => :build
```

Comment:
this is only needed for `:test`

Reply:
Thanks. Meanwhile, have switched to binutils as that is more common to be installed.